tests/unit/main/services/secureStorage.service.test.ts

```
@@ -0,0 +1,140 @@
+}));
+
+jest.mock('child_process', () => ({
+  execFile: (...args: unknown[]) => execFile(...args),
+}));
+
+jest.mock('../../../../src/main/services/mainDatabase.service', () => ({
+  __esModule: true,
+  default: {},
+}));
+
+import SecureStorageService from '../../../../src/main/services/secureStorage.service';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)